scripts/check_api_changes.ts

```diff
@@ -75,3 +75,14 @@ await Promise.all(
     console.log(`Validation of ${packageName} completed successfully`);
   })
 );
+
+const errors: Array<Error> = [];
+validationResults.forEach((result) => {
+  if (result.status === 'rejected') {
+    errors.push(result.reason);
+  }
+});
```

Comment:
fwiw. I tried `map.filter` and `.reduce` but they didn't yield simpler code due to result type diverging depending on `status` field.